using ocf_datapipes in Open-Source-Quartz-Solar-Forecast

roshnaeem opened the pull request:
# Pull Request

## Description
Importing [ocf_datapipes](https://github.com/openclimatefix/ocf_datapipes) `simple_pv_datapipe` and `pv_nwp_datapipe` to access the data pipes in the Open-Source-Quartz-Solar-Forecast project. 

> Note: The data files being used in `pv_config.yaml` haven't been uploaded in this PR, and have been added to the `.gitignore` file. 

Fixes  #30 and #73

peterdudfield commented on quartz_solar_forecast/datapipes/ocf_datapipe.py:
```diff
@@ -0,0 +1,45 @@
+import ocf_datapipes  # noqa
```
Could you add a comment at the top here about what the file does.

peterdudfield replied:
Can we make it really clear this is an example

peterdudfield commented:
Thank you @roshnaeem for getting this going. This is very helpful. I've left one comment

peterdudfield commented on quartz_solar_forecast/datapipes/pv_config.yaml:
```diff
@@ -0,0 +1,23 @@
+input_data:
```
should the name be `pv_and_nwp_config.yaml`?

roshnaeem replied:
yes, a good suggestion. @peterdudfield, I don't have to upload the data files, right? I added them in `.gitignore`. I can add the links of those files in the comments in the `.yaml` file if anyone wants to try them locally.

peterdudfield replied:
Yea I think that would be good. Then's it easy for other people to build on it

roshnaeem commented:
Hello @peterdudfield, I addressed the comments, kindly check. Moreover, I noticed in all files, functions are being used, so I refactored the code to keep it consistent with the project. 
Please let me know if more changes are needed. Also, if it is a good enough PR for GSOC, should I start working on the proposal?

peterdudfield commented on quartz_solar_forecast/datapipes/ocf_datapipe.py:
```diff
@@ -0,0 +1,56 @@
+import ocf_datapipes  # noqa
```
Could you add something at the top of this scrip saying what it does?

peterdudfield commented:
Looks really great, I wonder if you could move the files to `scripts` as its not quite part of the core library yet

roshnaeem commented:
> Looks really great, I wonder if you could move the files to `scripts` as its not quite part of the core library yet

Sure, I would move the directory to /scripts.

peterdudfield commented:
Thank you @roshnaeem for doing this
